Assets/MRTK/SDK/Features/Input/Handlers/ObjectManipulator.cs

```diff
@@ -408,6 +414,7 @@ private void Awake()
             moveLogic = new ManipulationMoveLogic();
             rotateLogic = new TwoHandRotateLogic();
             scaleLogic = new TwoHandScaleLogic();
+            smoothingLogic = (TransformSmoothingLogic)Activator.CreateInstance(transformSmoothingLogicType);
```

Comment:
Along the lines of if we add null checking below, should this be `as TransformSmoothingLogic` instead of a direct cast, so it returns null if the cast isn't possible instead of throwing an exception?

Reply:
Makes sense